budgie-app/src/screens/expenses/ExpenseList.tsx

```diff
@@ -0,0 +1,115 @@
+import React, {
+  FC, useCallback, useEffect, useState,
+} from 'react';
+import {
+  SafeAreaView, FlatList, RefreshControl, View, TouchableWithoutFeedback,
+} from 'react-native';
+
+import dayjs from 'dayjs';
+import tailwind from 'tailwind-rn';
+import { StackNavigationProp } from '@react-navigation/stack';
+import {
+  Button, Divider, Icon, Text,
+} from '@ui-kitten/components';
+import { useIsFocused } from '@react-navigation/native';
+import * as SplashScreen from 'expo-splash-screen';
+import { Header } from '../../components/Header';
+import { Expense } from '../../util/types';
+import { useToast } from '../../ToastProvider';
+import { useApi } from '../../hooks/use-request';
+import { ExpensesStackParamList } from '.';
+import { LOADING_INDICATOR_OFFSET } from '../../util/globals';
+
+const ExpenseItem: FC<{
+  item: Expense;
+  onPress: (id: string) => void
+}> = ({ item, onPress }) => (
+  <TouchableWithoutFeedback delayPressIn={0} onPress={() => onPress(item.id)}>
+    <View style={tailwind('mt-2')}>
+      <View style={tailwind('p-2 flex-row justify-between')}>
+        <View style={tailwind('flex-col ml-1')}>
+          <Text category="h5" status="primary" style={tailwind('font-bold')}>{item.category}</Text>
+          <Text appearance="hint">{item.name}</Text>
+        </View>
+        <View style={tailwind('flex-col justify-between mr-1')}>
+          <Text appearance="hint" style={tailwind('text-right')}>{dayjs(item.date).format('DD.MM.')}</Text>
```

Comment:
What if date was last year or the year before? -> show year if expense was not done in current year.